### PR TITLE
Ticket32792

### PR DESCRIPTION
**Pull request by teor2345**

### CODE REVIEW

File: tools/diagnostics.sh [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+#    Always show some network-wide diagnostics and authority diagnostics,
+#    regardless of the chosen node.
+#
+# Examples:
+#    tools/diagnostics.sh
+#    tools/diagnostics.sh 001a
+
+if [ ! -d "$CHUTNEY_PATH" ] || [ ! -x "$CHUTNEY_PATH/chutney" ]; then
+    # looks like a broken path: use the path to this tool instead
+    TOOLS_PATH=$(dirname "$0")
```
Comment (nmathewson) [inline tools/diagnostics.sh]: Should these be quoted?

File: tools/diagnostics.sh [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+case "$CHUTNEY_DATA_DIR" in
+  /*)
+    # if an absolute path, then leave as-is
+    # chutney will make this directory automatically if needed
+    ;;
+  *)
+    # if a relative path
+    if [ ! -d "$CHUTNEY_DATA_DIR" ]; then
+        # looks like a broken path: use the chutney path as a base
+        export CHUTNEY_DATA_DIR="$CHUTNEY_PATH/net"
```
Comment (nmathewson) [inline tools/diagnostics.sh]: Does this need to be on two lines for non-bash?

Comment (teor2345) [reply tools/diagnostics.sh]: I don't think so?
Shellcheck doesn't complain, and we do the same thing in test-network.sh.

File: tools/diagnostics.sh (hunk 1 above)
Comment (teor2345) [reply tools/diagnostics.sh]: I don't think so?
Shellcheck doesn't complain, and we do the same thing in test-network.sh.

**Pull request merged**